cmd/gobl/keygen.go

```diff
@@ -61,13 +63,8 @@ func homedir() (string, error) {
 	return user.HomeDir, nil
 }
 
-func defaultKeyfile() (string, error) {
-	const defaultFilename = ".gobl/id_es256.jwk"
-	home, err := homedir()
-	if err != nil {
-		return "", err
-	}
-	return filepath.Join(home, defaultFilename), nil
```

Comment:
I believe what these lines were doing was already implemented in `expandHome`, so I decided to use that function instead.

Reply:
Strange it wasn't standardised already... cool.